spec/request/swagger_spec.rb

```diff
@@ -610,6 +613,25 @@
       end
     end
 
+    describe 'bb' do
+      include BB::ClientHelpers
```

Comment:
Regarding DRYing up ClientHelpers - Rx/Sm/Bb require the Client from the Rx/Sm/Bb modules respectively - requiring passing in the module name to the call to the authenticated_client method. Moreover, the Rx::ClientHelper references the rx host setting unique to it. Drying these 3 client helpers may be more convoluted to than would warrant.